README.md

```diff
@@ -1,7 +1,28 @@
 [![Coverity Scan Build Status](https://scan.coverity.com/projects/5120/badge.svg)](https://scan.coverity.com/projects/5120)
 
-# Tvheadend HTSP PVR
-Tvheadend HTSP PVR client addon for [Kodi] (http://kodi.tv)
+# Tvheadend PVR addon for Kodi
+
+This is a [Kodi] (http://kodi.tv) PVR addon for connecting to a (tvheadend)[https://tvheadend.org] backend.
+
+## Build instructions
+
+### Linux
+
+The following instructions assume you check out all repositories in a directory named `Projects` in your home 
+directory. cmake doesn't recognize `~` which is why `$HOME` is used when referring to your home directory.
+
+In addition to standard build tools you need to have the following packages installed: `kodi-addon-dev kodi-pvr-dev`. 
+These packages are available from Kodi's PPA repositories.
+
+1. `cd ~/Projects`
```

Comment:
I'd drop this and the text about it. People should be capable of knowing where they want to put code if they're going to be building stuff.

Not that fussed, this is just my personal opinion.